robot/EDA/resources/locators.py

```diff
@@ -225,17 +231,17 @@
     "tab_menu": "//a[@title='{}']",
     "tab_tab": "//a[@title='Contacts']/../descendant::a[@title='{}']",
     "details_tab": "//div[contains(@class,'normal')]//span[@class='title' and text()='Details']",
-    "phone_home": "//span[text()='Home Phone']/../following-sibling::input",
+    "phone_home": "//span[text()='Phone']/../following-sibling::input",
     "run_cleanup": "//button[text()='Run Cleanup']",
     "phone_field": "//div//span[text()='Phone']/../following-sibling::div//span//span[text()='123-123-1234']",
     "phone_verify": "//div//span[text()='Phone']/../following-sibling::div//span//span[text()='123-123-1234']",
     "home_phone_verify": "//span[text()='Home Phone']/../following::div//span//span[text()='123-123-1234']",
     "copy_from": "//select[@class='contact-preferred-phone-picklist-input-select select uiInput uiInputSelect uiInput--default uiInput--select']",
-    "copy_from_home_phone": "//select[@class='contact-preferred-phone-picklist-input-select select uiInput uiInputSelect uiInput--default uiInput--select']//option[@value='Home Phone']",
+    "copy_from_home_phone": "//select[@class='contact-preferred-phone-picklist-input-select select uiInput uiInputSelect uiInput--default uiInput--select']//option[@value='Phone']",
     "successful_run": "//span[text()='The process was queued successfully. An email will be sent at the completion of the job.']",
     "enhanced_preferred_clear": "//div/span[text()='Enable Enhanced Preferred Phone Functionality:']/following::div[1]/div/div/label/span/img[@alt='False']",
     "enhanced_preferred_clear_faux": "//span[text()='Enable Enhanced Preferred Phone Functionality:']/following::div[1]/div/div/label/input/following::span[1]",
     "enhanced_preferred_set": "//span[text()='Enable Enhanced Preferred Phone Functionality:']/following::div[1]/div/div/label/span/img[@alt='True']",
     "enhanced_preferred_set_faux": "//span[text()='Enable Enhanced Preferred Phone Functionality:']/following::div[1]/div/div/label/input/following::span[1]",
-
+    "apex_jobs": "//a/mark[text()='{}']",
```

Comment:
This seems like it's a pretty generic locator. Could it be renamed something else and moved out of the contact locators? (The Apex Jobs page is a Salesforce Setup page, so could be navigated to from anywhere in Salesforce)

Reply:
It goes along with the ContactsPageObject model.

Reply:
got it, just thought it could be genericized more. But that's ok.